docs/setup.md

```diff
@@ -0,0 +1,151 @@
+# Setup
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed on your system:
+
+- Node.js (v18)
+- npm (v8.3.0 or higher)
+- [Docker](https://www.docker.com/products/docker-desktop/) (v4.32.0 or higher)
+
+## Developer Setup
+
+This guide will walk you through setting up a complete development environment, including Arranger and its complementary services.
+
+### Setting up supporting services
+
+We'll use our Conductor service, a flexible Docker Compose setup, to spin up Maestro's complementary services.
+
+1. Clone the Conductor repository and navigate to its directory:
+
+    ```bash
+    git clone https://github.com/overture-stack/conductor.git
+    cd conductor
+    ```
+
+2. Run the appropriate start command for your operating system:
+
+    | Operating System | Command |
+    |------------------|---------|
+    | Unix/macOS       | `make arrangerDev` |
+    | Windows          | `make.bat arrangerDev` |
+
+    <details>
+    <summary>**Click here for a detailed breakdown**</summary>
+
+    This command will set up all complementary services for Arranger development as follows:
+
+    ![arrangerDev](./assets/arrangerDev.svg 'Arranger Dev Environment')
+
+    | Service | Port | Description | Purpose in Arranger Development |
+    |---------|------|-------------|------------------------------|
+    | Conductor | `9204` | Orchestrates deployments and environment setups | Manages the overall development environment |
+    | Elasticsearch | `9200` | Distributed search and analytics engine | Provides fast and scalable search capabilities over indexed data |
+    | Stage | `3000` | Web Portal Scaffolding | Houses Arranger's search UI components |
+
+    :::note Arranger uses Elasticsearch 7 
+
+    Our search platform is built on and compatible with version 7.x of Elasticsearch. Applications and queries need to follow Elasticsearch 7 syntax and conventions.
```

Comment:
This notice was already in the overview, and while it's important to note they need to set up v7 here, using queries is not in the scope of "setup", strictly speaking.

I've amended this to match the changes I've suggested in the overview, but I'd advocate for removing this paragraph here.
```suggestion
    Our search platform is built on and compatible with version 7.x of Elasticsearch. All queries to ES must follow that version's syntax and conventions.
```